api/transactions/transactions.go

```diff
@@ -218,14 +218,14 @@ func (t *Transactions) Mount(root *mux.Router, pathPrefix string) {
 
 	sub.Path("").
 		Methods(http.MethodPost).
-		Name("transactions_send_tx").
+		Name("POST /transactions").
 		HandlerFunc(utils.WrapHandlerFunc(t.handleSendTransaction))
 	sub.Path("/{id}").
 		Methods(http.MethodGet).
-		Name("transactions_get_tx").
+		Name("GET /transactions/${id}").
 		HandlerFunc(utils.WrapHandlerFunc(t.handleGetTransactionByID))
 	sub.Path("/{id}/receipt").
 		Methods(http.MethodGet).
-		Name("transactions_get_receipt").
+		Name("GET /transactions/${id}/receipt").
```

Comment:
Here the variable is in the format `${id}` while [here](https://github.com/vechain/thor/blob/24b7f907b3cbec96c0dbfe75e4104a15bf373a43/api/blocks/blocks.go#L98) it's just `{revision}` without the dollar. I know it's just a string name but I think it's better to have a consistent naming convention

Reply:
Yepp, you're right, will change this